Donation-platform/src/main/java/com/google/model/Charity.java

```diff
@@ -15,31 +15,40 @@
 package com.google.model;
 
 import java.util.Collection;
-import com.google.appengine.api.datastore.Key;
-
+import com.googlecode.objectify.Key;
+import java.io.Serializable;
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Cache;
+import com.googlecode.objectify.annotation.Index;
+import java.lang.Long;
+
+@Entity
+@Cache
 /** Represents a Charity : id, name, link, categories, description, trendingScore. */
-public final class Charity implements Comparable<Charity> {
+public final class Charity implements Comparable<Charity>, Serializable {
 
   // Key id from datastore uniquely identifiying each charity.
-  private Key id;
+  @Id private Long id;
   // Name of charity
-  private String name;
+  @Index private String name;
```

Comment:
Will this automatically create an index for this field in datastore?

Reply:
Using objectify, I have to explicitly define the indexes. It is not created automatically without the annotation.